app/src/main/res/values/strings.xml

```diff
@@ -571,39 +571,37 @@
     <string name="please_read">PLEASE READ!</string>
     <string name="fasthub_faq_description">
         <![CDATA[
-        <h5>• Why can\'t I see my <b>Organizations</b> either <i>Private</i> or <i>Public</i> ones?</h5>
-        <p>Open up https://github.com/settings/applications and look for FastHub, open it then scroll to Organization access and click on Grant Button,
-        alternatively login via <b>Access Token</b> which will ease this setup.</p>
+        <h5>• Why can I see neither my <i>Private</i> nor my <i>Public</i> <b>Organizations</b>?</h5>
+        <p>Open up https://github.com/settings/applications and look for FastHub, open it up, scroll down to "Organization access" and click on the "Grant" button.
+        Alternatively you can sign in using <b>Access Token</b> which will ease the process.</p>
 
-        <h5>• I tried to login via Access Token &amp; OTP but it does not work?</h5>
-        <p>You can\'t login via Access Token &amp; OTP all together due to the lifetime of the OTP code, you\'ll be required to login in every few seconds.</p>
+        <h5>• I have tried signing in with Access Token and One-Time Password (OTP) without any luck. What's wrong?</h5>
+        <p>You can\'t sign in using Access Token and OTP all together. Due to the limited lifetime of such passwords you\'ll have to repeat a sign in procedure every few seconds.</p>
 
-        <h5>• Why are my Private Repo &amp; Enterprise Wiki not showing up?</h5>
-        <p>It\'s due to FastHub scraping GitHub Wiki page &amp; Private Repos require session token that FastHub is unable to obtain.</p>
```

Comment:
aren't not isn't

Reply:
`Isn't` is correct here, because we're meaning only one Wiki here.  
`Repository's Wiki`

Reply:
Nope you are saying private OR enterprise = 2